sandbox

```diff
@@ -169,6 +169,25 @@ sandbox () {
     docker-compose -f "$SANDBOX_DIR/docker-compose.yml" "$@"
   }
 
+  dc_no_cache () {
+    docker-compose build --no-cache
+    dc "$@"
+  }
+
+  dc_with_clean_check () {
+    CLEAN_FILE=.clean
```

Comment:
sandbox may not be run in the correct folder, the file must be

`CLEAN_FILE="$SANDBOX_DIR/.clean"`